panel/src/main/java/net/skyscanner/backpack/panel/BpkPanel.kt

```diff
@@ -2,18 +2,19 @@ package net.skyscanner.backpack.panel
 
 import android.content.Context
 import android.support.annotation.Dimension
+import android.support.annotation.Nullable
 import android.support.v7.widget.LinearLayoutCompat
 import android.util.AttributeSet
 import android.widget.LinearLayout
 
 
-class BpkPanel(
+open class BpkPanel(
```

Comment:
open might be needed in the text component as well

Reply:
It was missing only here :)